parameterspace/priors/base.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, bounds: Tuple):
         Args:
             bounds: Lower and upper bound of the prior.
         """
+        self._init_args: Tuple
```

Comment:
You are correct, it is considered bad practice to use a class/instance attribute which not have been declared in `__init__` or as class attribute, that what pylint complains about here.

To silent the warning, yes, defining it as class attribute would be fine, too. But I assumed that `_init_args` and `_init_kwargs` are no class attributes, but instance attributes, correct? Therefor, it would be misleading to type them as class attributes.

The interpreter just ignores that type declaration and therefore still would treat those two variables as instance attributes, even if they were typed like a class attribute.

Reply:
PS: I shortly tried to assign empty defaults to both of them, as it seemed cleaner to me (because some methods expect the existence of those two), but then the unit tests failed horribly (I think it broke some wrapper/factory logic). Didn't want to spent too much time on it, that's why just typing them here.

Reply:
Thanks for the explanation. You are correct, those are indeed instance attributes, and the way they are assigned is probably also bad practice, as they are not consistently created.